src/apps/Altinn.AccessManagement/src/Altinn.AccessManagement.Api.Internal/Controllers/Bff/ConsentController.cs

```diff
@@ -66,6 +68,46 @@ public async Task<IActionResult> GetConsentRequest([FromRoute] Guid requestId, C
             return Forbid();
         }
 
+        [HttpGet]
+        [Authorize(Policy = AuthzConstants.SCOPE_PORTAL_ENDUSER)]
+        [Route("consentrequests/list/{partyUuid}", Name = "requestlist")]
+        public async Task<IActionResult> RequestList([FromRoute] Guid partyUuid, CancellationToken cancellationToken = default)
+        {
+            Guid? performedBy = UserUtil.GetUserUuid(User);
+            if (performedBy == null)
+            {
+                return Unauthorized();
+            }
+
+            Core.Models.Consent.ConsentPartyUrn performedByParty = Core.Models.Consent.ConsentPartyUrn.PartyUuid.Create(performedBy.Value);
+
+            Result<List<ConsentRequestDetails>> consentRequest = await ConsentService.GetRequestsForParty(partyUuid, true, cancellationToken);
+            if (consentRequest.IsProblem)
+            {
+                return consentRequest.Problem.ToActionResult();
+            }
+
+            List<ConsentRequestDetails> authorizedRequests = new List<ConsentRequestDetails>();
+
+            // Check if the user is authorized to view the consent request. Anyone with read access to access management can view the consent request details.
+            if (consentRequest.Value.Count > 0)
+            {
+                foreach (var req in consentRequest.Value)
+                {
+                    if (req.From.IsPartyUuid(out Guid resourePartUuuid))
+                    {
+                        bool isAuthorized = await AuthorizeResourceAccess(accessManagementResource, resourePartUuuid, User, "read");
```

Comment:
```suggestion
                        bool isAuthorized = await AuthorizeResourceAccess(accessManagementResource, resourcePartyUuid, User, "read");
```

kan resourcePartyUuidnoen gang være forskjellig fra input partyUuid?

Eller hadde det vært nok å autorisere en gang, før man i det hele tatt henter ConsentService.GetRequestsForParty?